### ISSUE TITLE
Is this still operational?

### ISSUE THREAD

**Issue by trfetzer** [jose-donato/ollama-reply#5]
I was playing with it but could not get it to run on my machine.

**Comment by jose-donato** [comment]
> I was playing with it but could not get it to run on my machine.

tbh haven't used it for a while 😅

do you have ollama with the model running?

let me try and I'll come back to you

**Comment by trfetzer** [comment]
Yes, I have. It just doesnt show the items. I use Ollama as a local LLM and
interact in it with python. I first didnt have LLAMA3:8b, but 3.1:72b but I
pulled the correct one and then re-installed the extension, but its still
not running unfortunately.

Best Thiemo
